docs/website/docs/examples/qdrant_zendesk/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Similarity Searching with dlt and Qdrant
+description: Learn how to use the dlt source, Zendesk and dlt destination, Qdrant to conduct a similarity search on your tickets data.
+keywords: [similarity search, example]
+---
+import Header from '../_examples-qdrant-header.md';
+
+<Header
+    intro="In this tutorial, you will learn how to do use dlt to store your
+    vectorized Zendesk tickets data in the dlt destination; Qdrant. You can
+    use Qdrant's vectorization and similarity searching capabilities on your tickets data,
+    while using dlt as a medium to automate your pipeline."
+    slug="qdrant_zendesk"
+    run_file="qdrant-snippets"
+/>
+
+Qdrant has been introduced as a new [destination](https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant) for dlt, adding onto our list of high-performance vector databases. Similar to our guide on using the [Zendesk source to extract ticket data and store it into weaviate](https://dlthub.com/docs/walkthroughs/zendesk-weaviate#configuration), this article drafts a similar system to map the vectorized ticket data from Zendesk to Qdrant!
+
+## Importing Ticket Data from Zendesk API to Qdrant
+
+In order to begin the import, we must setup first. This means making sure we have the correct configuration and environment to start working. To do so, we first install dlt with Qdrant support:
+
+```commandline
+pip install dlt[qdrant]
+```
+
+Next, install the dlt Zendesk verified source:
+
+```commandline
+dlt init zendesk qdrant
+```
+
+Now, configure the destination credentials for Qdrant in .dlt/secrets.toml as follows:
+
+```commandline
```

Comment:
here you can link https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant instead of all this text